src/minimega/container.go

```diff
@@ -574,26 +574,14 @@ func NewContainer(name string) *ContainerVM {
 }
 
 func (vm *ContainerVM) Launch(ack chan int) error {
-	go vm.launch(ack)
+	vm.asyncLaunch(ack)
```

Comment:
The call stack becomes

*ContainerVM.Launch() -> *ContainerVM.asyncLaunch() -> *ContainerVM.launch()

The top two launch wrappers don't appear to do much of anything, can this be collapsed here and in kvm or is there a reason to have 3 layers of launch indirection that I'm missing?